src/routes/generate-outfit/+page.svelte

```diff
@@ -2,27 +2,53 @@
     import Header from '$lib/components/Header/Header.svelte';
     import Checkbox from '$lib/components/Checkbox/Checkbox.svelte';
     import Button from '$lib/components/Button/Button.svelte';
```

Comment:
Can you add a link to the home page for this page. Just an anchor is fine for now

Reply:
added link